### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by tbm**

### CODE REVIEW

File: nikola/conf.py.in [hunk 1]
```diff
@@ -967,7 +967,7 @@ PRETTY_URLS = ${PRETTY_URLS}
 # If True, use the scheduling rule to all posts (not pages!) by default
 # SCHEDULE_ALL = False
 
-# Do you want a add a Mathjax config file?
+# Do you want an add a Mathjax config file?
```
Comment (Kwpolska) [inline nikola/conf.py.in:970]: to*

File: CHANGES.txt [hunk 2]
```diff
@@ -1193,7 +1193,7 @@ Features
   configuring the section pages (Issue #1980)
 * For themers: Each ``post`` are now asssociated with section_color,
   section_link, and section_name (Issue #1980)
-* Each new section page has a auto-assigned color based on shifting
+* Each new section page has an auto-assigned color based on shifting
```
Comment (Kwpolska) [inline CHANGES.txt:1196]: Let’s not bother modifying an ancient changelog line.

File: nikola/data/samplesite/pages/dr-nikolas-vendetta.rst [hunk 3]
```diff
@@ -360,7 +360,7 @@ once more to its former business-like tone.
 "I am not going to tell you very much about it. But from what I do tell
 you, you will be able to gather a great deal and imagine the rest. To
 begin with, there is a man living in this world to-day who has done me a
-great and lasting injury. What that injury is is no concern of yours.
+great and lasting injury. What that injury is no concern of yours.
```
Comment (Kwpolska) [inline nikola/data/samplesite/pages/dr-nikolas-vendetta.rst:363]: It’s a book from Project Gutenberg, there’s no point in modifying it here, and this typo might exist in the original book.

File: nikola/data/samplesite/pages/quickstart.rst [hunk 4]
```diff
@@ -363,7 +363,7 @@ Images
 
 __ ../quickref/#directives
 
-To include an image in your document, you use the the ``image`` directive__.
+To include an image in your document, you use the ``image`` directive__.
```
Comment (Kwpolska) [inline nikola/data/samplesite/pages/quickstart.rst:366]: This is also a third-party document, this typo fix might be reported upstream if you wish.

File: nikola/data/samplesite/pages/dr-nikolas-vendetta.rst (hunk 3 above)
Comment (Kwpolska) [reply nikola/data/samplesite/pages/dr-nikolas-vendetta.rst]: And actually, this is *not* a typo.

File: nikola/conf.py.in (hunk 1 above)
Comment (tbm) [reply nikola/conf.py.in]: Uh sorry about that.

File: CHANGES.txt (hunk 2 above)
Comment (tbm) [reply CHANGES.txt]: ok

File: nikola/data/samplesite/pages/dr-nikolas-vendetta.rst (hunk 3 above)
Comment (tbm) [reply nikola/data/samplesite/pages/dr-nikolas-vendetta.rst]: ok

File: nikola/data/samplesite/pages/quickstart.rst (hunk 4 above)
Comment (tbm) [reply nikola/data/samplesite/pages/quickstart.rst]: ok will do

**Comment by tbm** [comment]
sorry for creating more work than the typo is worth.  Anyway, latest version should be ok.

**Comment by Kwpolska** [comment]
No problem, I’ll fix our copy of quickstart since we modify it slightly.

**Pull request merged**